meter.go

```diff
@@ -5,6 +5,10 @@ import (
 	"time"
 )
 
+// Meters are replaced by RateCounter as of Jul 2020 because of their exponential weighted nature that messes up MonViz.
+// As such we turn off their external use to force usage of RateCounter in the service repo.
+// Then next step is to actually use a mix of RateCounter and Meter for better efficiency.
```

Comment:
Not fully sure I understand this last comment. So we do want to use Meter in the future? If so, why are we forbidding external user of meter?

Reply:
this is the thread, just an optimization thing
https://twitter.slack.com/archives/C9LJM2ERM/p1594403831173100?thread_ts=1594335819.161200&cid=C9LJM2ERM

Reply:
so basically we want to stop people using meter, but not deleting the file yet
we move to rate counter (it will help monviz)
once that's done will (maybe) make an optimized version of rate counter (so it would be as optimized as meter but with the guarantees of rate counter, that's what I meant by "a mix of")